calendar/Module.md

```diff
@@ -0,0 +1,102 @@
+## Overview
+The module provides the capability to manage events and calendar operations and supports service account authorization that can provide delegated domain-wide access to GSuite domain and support admin to do operations on behalf of the domain users.
+
+This module supports [Google Calendar API](https://developers.google.com/calendar/api) version V3.
+ 
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [Google account](https://accounts.google.com/signup/v2/webcreateaccount?utm_source=ga-ob-search&utm_medium=google-account&flowName=GlifWebSignIn&flowEntry=SignUp)
+- Obtain tokens - Follow [this link](https://developers.google.com/identity/protocols/oauth2)
```

Comment:
Obtain Tokens
Follow the steps [here](https://developers.google.com/identity/protocols/oauth2) to obtain tokens.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884